mappings/ce3d4ec01b3a79fd855e21bdacc1aa5b1bd7de49e27e47d3286223db4d454f575a.json

```diff
@@ -0,0 +1,64 @@
+{
+    "subject": "ce3d4ec01b3a79fd855e21bdacc1aa5b1bd7de49e27e47d3286223db4d454f575a",
+    "url": {
+        "sequenceNumber": 0,
+        "value": "https://www.meowz.xyz",
```

Comment:
Hi @orthogonalistic,

if your site is still not up, you can update this value, once it is running. The URL value is not required for the submission. For now, we can't verify the URL. Please, update the PR without URL value or with a valid page.

Thanks!

Reply:
Thank you for the reply. The website is now up. Thanks again for the quick reply 😺